WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasDisplayButton.lua

```diff
@@ -420,6 +449,67 @@ local methods = {
             end
         end
 
+        function self.callbacks.OnDragStart()
+            if WeakAuras.IsImporting() or not data.parent then return end;
+            WeakAuras.PickDisplay(data.id)
+            WeakAuras.SetDragging(data)
+            WeakAuras.UpdateButtonsScroll()
+        end
+
+        function self.callbacks.OnDragStop()
+            if self.frame.isDragged then
+                WeakAuras.SetDragging()
+                Show_DropIndicator()
+
+                self.frame:SetScript("OnUpdate", nil)
+
+                self.frame.isDragged = false
+                self.frame:StopMovingOrSizing()
+                -- show controls
+                self.downgroup:Show()
+                self.loaded:Show()
+                self.ungroup:Show()
+                self.upgroup:Show()
+                self.view:Show()
+
+                local id, parent = data.id, data.parent
+                local parentData = WeakAuras.GetData(parent)
+                local children = parentData.controlledChildren
+
```

Comment:
I need to map out the logic for that.  
right now i just check that there is a parent which is the easy way :smile:  
In my eyes this should include to be able to put a ungrouped button into a group and vice versa.
This means there needs to be a visual hint (and the logic) what operation would performed on dropping.